submittal/src/main/java/com/kyriba/submittal/domain/dto/SpeakerDataDto.java

```diff
@@ -0,0 +1,31 @@
+package com.kyriba.submittal.domain.dto;
+
+/**
+ * @author M-ABL
+ */
+public class SpeakerDataDto<T>

```

Comment:
Very strange class

Reply:
the idea is to return SpeakerId and some extra data

Reply:
Usually DTOs are created as simple as possible and composition with generics is not used (tools work bad, not clear code, etc.).